excel-importer/src/main/java/uk/gov/hmcts/ccd/definition/store/excel/parser/RoleToAccessProfilesParser.java

```diff
@@ -16,15 +17,15 @@
 
 public class RoleToAccessProfilesParser {
 
-    public List<RoleToAccessProfilesEntity> parse(Map<String, DefinitionSheet> definitionSheets,
-                                                  ParseContext parseContext) {
+    public List<RoleToAccessProfilesEntity> parse(final Map<String, DefinitionSheet> definitionSheets,
+                                                  final ParseContext parseContext) {
         try {
             final List<DefinitionDataItem> roleToAccessProfiles = definitionSheets
-                .get(SheetName.ROLE_TO_ACCESS_PROFILES.getName()).getDataItems();
-            return roleToAccessProfiles
-                .stream()
-                .map(questionItem ->
-                    createRoleToAccessProfileEntity(parseContext, questionItem)).collect(Collectors.toList());
+                .get(SheetName.ROLE_TO_ACCESS_PROFILES.getName())
+                .getDataItems();
+            return roleToAccessProfiles.stream()
+                .map(questionItem -> createRoleToAccessProfileEntity(parseContext, questionItem))
```

Comment:
```suggestion
                .map(roleToAccessProfile -> createRoleToAccessProfileEntity(parseContext, roleToAccessProfile))
```
Not related to your changes as it was already (incorrectly) named `questionItem`, but just spotted this since this line is being changed as part of this PR - may be worth updating to make sense while we're here!

Reply:
Actioned in e2aa9add04f3f367beedea20683c03decc7a5716